.github/CONTRIBUTING.md

```diff
@@ -24,7 +24,46 @@ Also, try to include **steps to reproduce** the issue and expand on the **actual
 
 If you would like to contribute to Parity Ethereum, please **fork it**, fix bugs or implement features, and [propose a pull request](https://github.com/paritytech/parity-ethereum/compare).
 
-Please, refer to the [Coding Guide](https://wiki.parity.io/Coding-guide) in our wiki for more details about hacking on Parity.
+### Labels & Milestones
+
+We use [labels](https://github.com/paritytech/parity-ethereum/labels) to manage PRs and issues and communicate state of a PR. Please familiarize yourself with them. Furthermore we are organizing issues in [milestones](https://github.com/paritytech/parity-ethereum/milestones). Best way to get started is to a pick a ticket from the current milestone tagged [`easy`](https://github.com/paritytech/parity-ethereum/labels/Q2-easy%20%F0%9F%92%83) and get going or [`mentor`](https://github.com/paritytech/parity-ethereum/labels/Q1-mentor%20%F0%9F%95%BA) and get in contact with the mentor offering their support on that larger task.
+
+### Rules
+
+There are a few basic ground-rules for contributors (including the maintainer(s) of the project):
+
+* **No `--force` pushes** or modifying the master branch history in any way. If you need to rebase, ensure you do it in your own repo.
+* **Non-master branches**, prefixed with a short name moniker (e.g. `gav-my-feature`) must be used for ongoing work, and include the associated issue ID (if any) in the branch name.
+* **All modifications** must be made in a **pull-request** to solicit feedback from other contributors.
+* A pull-request *must not be merged until CI* has finished successfully.
+* Contributors should adhere to the [house coding style](https://wiki.parity.io/Coding-guide).
```

Comment:
This isn't up to date. I think we should either be linking to the Substrate one, or update this one to match

Reply:
Let's update the wiki to match the substrate one :+1:

Reply:
https://github.com/paritytech/wiki/pull/332 just copy-paste?

Reply:
I think a potential issue is that we would have the same coding style guide stored in 3x places, each with the same contents:

- Style Guide for Rust in Polkadot - https://github.com/w3f/polkadot-wiki/blob/master/docs/polkadot/build/rust-style-guide.md, which is published at http://wiki.polkadot.network/en/latest/polkadot/build/rust-style-guide/
- Style Guide for Rust in Substrate - https://github.com/paritytech/wiki/blob/master/Substrate-Style-Guide.md, which is published at https://wiki.parity.io/Substrate-Style-Guide.html
- Style Guide for Rust in Parity Ethereum - https://github.com/paritytech/wiki/pull/332, which will be published at https://wiki.parity.io/Coding-guide

Reply:
I think https://wiki.parity.io/Coding-guide should contain a coding style guide that's "general" across all Parity products, and then the https://wiki.parity.io/Substrate-Style-Guide.html would extend it with Substrate-specific aspects, and link to the "general" guide (as with http://wiki.polkadot.network/en/latest/polkadot/build/rust-style-guide/), and that we should add an additional https://wiki.parity.io/Parity-Ethereum-Style-Guide.html link that extents the "general" guide with Parity Ethereum-specific aspects.

Reply:
@HCastano i've pushed a commit that updates the link so it points to https://wiki.parity.io/Parity-Ethereum-Style-Guide, which works since https://github.com/paritytech/wiki/pull/332 has been merged